framework/src/utils/PetscSupport.C

```diff
@@ -632,6 +632,26 @@ getCommonPetscKeys()
     "-snes_ls -snes_max_it -snes_rtol -snes_type -sub_ksp_type -sub_pc_type", "", true);
 }
 
+void
+setSinglePetscOption(const char * name, const char * value)
```

Comment:
Just kidding, I will change the wrapper to take std::strings with a default value for the second arg.